app/screens/setting/EthSetting.tsx

```diff
@@ -4,6 +4,8 @@ import styles from "../../styles";
 import Button from "../../components/Button";
 import { GetEthKeypair } from "../../helpers/setting/EthKeypair";
 import AsyncStorage from "@react-native-async-storage/async-storage";
+import { ethers } from "ethers";
+import * as Random from "expo-random";
```

Comment:
Might want to use a better lib such as https://www.npmjs.com/package/react-native-securerandom

Reply:
should be secure enough https://docs.expo.dev/versions/latest/sdk/random/.
expo-random provides a native interface for creating strong random bytes. With Random you can create values equivalent to Node.js core crypto.randomBytes API. expo-random also works with expo-standard-web-crypto, which implements the W3C Crypto API for generating random bytes.